src/components/Filters/FilterChecklist.vue

```diff
@@ -201,22 +218,28 @@ label {
 
 .license-help {
   cursor: pointer;
+  padding: 0;
 }
 
 .filter-checkbox-wrapper {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  margin-top: 0.5rem;
-  padding-bottom: 0.75rem;
+  margin-top: 0.8rem;
 }
 
 .checkbox {
   display: flex;
   align-items: center;
   font-size: 0.875rem;
+  font-weight: 500;
+}
+.checkbox input {
+  width: 20px;
+  height: 20px;
```

Comment:
```suggestion
  width: 1.25rem;
  height: 1.25rem;
```

https://nekocalc.com/px-to-rem-converter